_episodes/07-tasks-and-automation.md

```diff
@@ -1,19 +1,114 @@
+- Rename Group: This will allow you to rename a group.
+
+The difference between these two actions lists might seem confusing at first. Task actions allow you to work with a particular task in terms of creating and managing one specific task at a time. The Manage Existing Tasks menu allows you to create and manage groups, exporting and importing, and assigning keyboard shortcuts to tasks.
+
+Tasks don't have to be categorized in groups or be linked to keyboard shortcuts. This is up to the users and their preference for how they organize their task workspace.
+
+## Create and a new task
+To create a task in the MarcEditor, Go to Tools in the top level menu and then Manage Tasks. In the Task Actions menu, select "New Task". This opens the task list window where you will see the name you gave your task, a description, tasks, and actions.
+
+Please note that you will see the name of the task and can type something different in this window. However, these changes only for the name won't take effect. To change the name of a task, you need to go to Task Actions and select Rename task.
+
+You can add or update a description in this task list window. The box in the middle will list the fuctions (or tasks) for this particular task. If there are none, this window will be empty.
+
+Actions include:
+- Add: Add a new function or task
+- Duplicate: Clone a function or task already in the list
+- Delete: Delete a function or task
+- Edit: change an existing function or task
+
+You can add a new task by clicking on the plus button. You can add any type of function available in the MarcEditor Tools and Edit Shortcuts menu options. This includes the RDA Helper, build a new field, change the case of a MARC field, or generate cutters. You can even include a list of functions (tasks) from another Task.
+
+Click on save at the left bottom corner of the Task List window to save your Task. From there you can close the Task Manager window. To run a task, go to Tools, then Assigned Taks, Current Tasks.
+
+If you have a lot of Tasks, it does help to assign them to groups to more easily find what you need. If you don't use Groups, the Task Manager will list all your Tasks in alphabetical order and it might be difficult to see them on especially on small screens such as laptops.
+
+>## Create and run a new Task that runs the RDA helper and removes 9XX fields
+>
+>1. .
+>
+> > ## Solution
+> > 1. Go to Tools in the upper menu in the MarcEditor
+> > 2. Select Manage Tasks
+> > 3. In the Task Manager window, in Task Actions, select New Task
+> > 4. Click Select.
+> > 5. Name your Task
+> > 6. In the Task List window, you can describe this task.
+> > 7. Click on the plus button and add RDA Helper.
+> > 8. Click on the plus button and add a new Add Delete Task Field
+> > 9. In the new window to delete a field, add 9XX and click on Delete field.
+> > 10. Save Task
+> > 11. Close the Task Manager
+> > 12. Go to Tools, then Assigned Tasks, and select your task that you just created
```

Comment:
```suggestion
> > 12. Go to Tools >> Assigned Tasks >> Currently Available Tasks and select your newly created task.
```

Reply:
I think it might be useful to use arrows instead of words (ex. then) to indicate paths. This will be something to review/decide on for consistency across the lesson.